hack/clibyexample/template

```diff
@@ -0,0 +1,12 @@
+{{range .items}}
```

Comment:
TOC added. Titles are now the command full name, descriptions are simple text below the title.